src/cf-layout/src/molecules/heroes.less

```diff
@@ -62,133 +65,121 @@
-            border-bottom-width: 1px;
-        }
-
-        .respond-to-min( @bp-sm-min, {
-            margin-bottom: 0;
+        .respond-to-max( @bp-sm-max, {
+            font-size: @size-iv;
         } );
     }
 
     &_image-wrapper {
         .grid_column( 1, 1 );
 
+        overflow: hidden;
+
         .respond-to-min( @bp-sm-min, {
             .grid_column( 5, 12 );
 
-            // Keep the image vertically centered
+
+            // IE specific fix
             display: table-cell;
             vertical-align: middle;
+
+            @supports ( display: flex ) {
+                display: block;
+                margin: auto;
+            }
         } );
 
         .respond-to-max( @bp-xs-max, {
             margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
         } );
+    }
 
-        &__bleed-vertical {
-            position: relative;
+    &_image {
+        .u-flexible-container-mixin( 16, 9 );
 
-            .m-hero_image {
-                display: block;
-                height: auto;
-            }
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: contain;
+    }
 
-            .m-hero_image__bleeding {
-                width: 100%;
-                height: 100%;
-                position: absolute;
-                top: 0;
-                background-position: center;
-                background-repeat: no-repeat;
-                background-size: cover;
+    &__knockout {
+        background-color: @hero-knockout-bg;
+        color: @hero-knockout-text;
+    }
+
+    &__bleeding {
+        .respond-to-min( @bp-sm-min, {
+
+            .m-hero_wrapper {
+                position: relative;
             }
 
-            .respond-to-max( @bp-xs-max, {
-                .m-hero_image__bleeding {
-                    display: none;
-                }
-            } );
+            .m-hero_image-wrapper {
+                margin-top: unit( @grid_gutter-width / @base-font-size-px, em ) * -1;
+                margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em ) * -1;
 
-            .respond-to-min( @bp-sm-min, {
-                .m-hero_image {
-                    display: none;
-                }
-            } );
+                //IE specific fix
+                position: absolute;
+                height: 100%;
 
-            .respond-to-min( @bp-med-min, {
-                .m-hero_image {
-                    display: none;
+                @supports ( display: flex ) {
+                    position: relative;
+                    height: initial;
                 }
-            } );
-        }
-    }
+            }
 
-    &_image {
-        // height is the same as min-height in a table
-        height: unit( @hero-image-height / @base-font-size-px, em );
+            .m-hero_image {
+                // Remove the flexible container padding so that the image is a consistent width
+                padding-bottom: 0 !important;
```

Comment:
Discussed in chat. Because the images are being loaded through inline styles, the padding must be recalculated if it's not a standard sized image, and that inline style has a higher specificity. 

I had originally absolutely positioned the flexible container within the image wrapper, but that proved far more complex due to the height of the image changing as you reduced the browser. So something like this didn't work unless the browser was at full width:

```
  top: 50%;
  margin: -(half the image height);
```

It turned out to be far simpler to remove the flexible container padding and making the image 100% in height and width of the wrapper.